config/config.json

```diff
@@ -2,7 +2,7 @@
   "development": {
     "username": "root",
     "password": null,
```

Comment:
Not sure what password or other attribute stuff would need to be for heroku db stuff